contrib/tut/src/mill/contrib/tut/TutModule.scala

```diff
@@ -0,0 +1,44 @@
+package mill
+package contrib.tut
+
+import ammonite.ops.Path
+import coursier.MavenRepository
+import java.lang.reflect.Method
+import java.net.URLClassLoader
+import mill.scalalib._
+import mill.util._
+import scala.util.matching.Regex
+
+trait TutModule extends ScalaModule {
+  def tutSourceDirectory = T.sources { millSourcePath / 'tut }
+  def tutTargetDirectory: T[Path] = T { T.ctx().dest }
+  def tutClasspath: T[Agg[PathRef]] = tutJar() ++ compileClasspath()
+  def tutScalacPluginIvyDeps: T[Agg[Dep]] = scalacPluginIvyDeps()
+  def tutNameFilter: T[Regex] = T { """.*\.(md|markdown|txt|htm|html)""".r }
+  def tutScalacOptions: T[Seq[String]] = scalacOptions()
+  def tutVersion: T[String] = "0.6.7"
+
+  def tutJar: T[Agg[PathRef]] = T {
+    Lib.resolveDependencies(
+      repositories :+ MavenRepository(s"https://dl.bintray.com/tpolecat/maven"),
+      Lib.depToDependency(_, "2.12.4"),
```

Comment:
Is there a "millScalaVersion" somewhere that we can use for definitions like this?

Reply:
Actually I don't think that makes sense here - we want Tut to run with the `scalaVersion` of the module